docker/resource_docker_volume_funcs.go

```diff
@@ -0,0 +1,89 @@
+package docker
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	dc "github.com/fsouza/go-dockerclient"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceDockerVolumeCreate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
I would rather these functions remain in the main `resource_docker_volume.go` file. Ideally we minimize the number of files a given developer needs to be familiar with when working on any provider

Reply:
I'd like them separated to have the same structure among all `resources`. IMHO this makes life easier for developers and more structured.

Reply:
I can see your point, I was unaware that this Provider already had resources with their functions split out. 

Unfortunately, that means I would rather see _all_ the resources combined into 1 file per resource (excluding `_test.go` file of course). That would be more consistent with Terraform's Provider style, meaning thus far "all" Terraform Provider Resources are contained in 1 file per resource. 

I also do not share your opinion that splitting things into 2 files makes anything easier. Instead of having to look in one place for schema and CRUD functions, I now must look in two.

Reply:
Ok, I can live with your proposal as well. So I'll migrate it back. Nevertheless, the other changes with the `validators` are fine right?